scripts/spack/packages/umpire/package.py

```diff
@@ -58,13 +55,10 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
     patch('camp_target_umpire_3.0.0.patch', when='@3.0.0')
     patch('cmake_version_check.patch', when='@4.1')
     patch('missing_header_for_numeric_limits.patch', when='@4.1:5.0.1')
-    # TRIBOL EDIT BEGIN
-    patch('cmake_devicealloc_option_6.0.0serac.patch', when='@6.0.0tribol')
-    # TRIBOL EDIT END
 
     # export targets when building pre-6.0.0 release with BLT 0.4.0+
-    patch('https://github.com/LLNL/Umpire/commit/5773ce9af88952c8d23f9bcdcb2e503ceda40763.patch',
-          sha256='f5c691752e4833a936bce224bbe0fe884d3afa84c5e5a4a481f59a12840159c9',
+    patch('https://github.com/LLNL/Umpire/commit/5773ce9af88952c8d23f9bcdcb2e503ceda40763.patch?full_index=1',
+          sha256='f3b21335ce5cf9c0fecc852a94dfec90fb5703032ac97f9fee104af9408d8899',
```

Comment:
I don't know the background pertaining to this or the particular sha, but I trust this is correct.

Reply:
I grabbed it from Serac's package.py from Umpire and looks like it matches what's in the spack repo.  See [here](https://github.com/LLNL/serac/blob/develop/scripts/spack/packages/umpire/package.py) and [here](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/umpire/package.py).